cmd/configmaps.go

```diff
@@ -58,42 +65,49 @@ func validateConfigMapCommandInput(args []string) error {
 	return nil
 }
 
-func executeConfigMapCleanupCommand(args []string) error {
-	if len(config.Resource.Labels) == 0 {
-		if err := listConfigMaps(); err != nil {
+func executeConfigMapCleanupCommand(resources resource.ConfigMapResources, config *cfg.Configuration) error {
+	namespace := resources.GetNamespace()
+	labels := config.Resource.Labels
+	if len(labels) == 0 {
+		configMapNames, labels, err := resources.ListConfigMaps()
+		if err != nil {
 			return err
 		}
+		log.WithFields(log.Fields{
+			"\n - namespace":    namespace,
+			"\n - 🔓 configMaps": configMapNames,
+			"\n - 🎫 labels":     labels,
+		}).Info("Please use labels to select ConfigMaps. The following ConfigMaps and Labels are available:")
 		return nil
 	}
 
-	c := config.Resource
-	namespace := config.Namespace
-
 	log.WithField("namespace", namespace).Debug("Looking for ConfigMaps")
 
-	foundConfigMaps, err := openshift.ListConfigMaps(namespace, getListOptions(c.Labels))
+	err := resources.LoadConfigMaps(getListOptions(labels))
 	if err != nil {
-		return fmt.Errorf("Could not retrieve ConfigMaps with labels '%s' for '%s': %w", c.Labels, namespace, err)
+		return fmt.Errorf("could not retrieve ConfigMaps with labels '%s' for '%s': %w", labels, namespace, err)
 	}
 
-	unusedConfigMaps, err := openshift.ListUnusedResources(namespace, foundConfigMaps)
+	err = resources.FilterUsed()
 	if err != nil {
-		return fmt.Errorf("Could not retrieve unused ConfigMaps for '%s': %w", namespace, err)
+		return fmt.Errorf("could not retrieve unused configMaps for '%s': %w", namespace, err)
```

Comment:
"ConfigMaps" seems to be written with a capital initial letter in all other locations (usage screen, debug output). It might make sense to keep it this way here, too?